rfcs/009-exunit.md

```diff
@@ -0,0 +1,129 @@
+---
+name: Formal RFC
+about: Submit a formal Request For Comments for consideration by the team.
+title: 'Use ExUnit testing framework for unit testing'
+labels: rfc, discussion
+assignees: ''
+
+---
+
+# Introduction
+
+With the upgrade of supported Erlang version and introduction of Elixir into our
```

Comment:
I think just this paragraph should be the introduction. Then the section below should be below the abstract as an actual reason why we want to move to exunit.